packages/ckeditor5-engine/src/view/selection.ts

```diff
@@ -759,15 +714,5 @@ export type ViewSelectionChangeEvent = {
  * An entity that is used to set selection.
  *
  * See also {@link module:engine/view/selection~Selection#setTo}
- *
- * @typedef {
- *    module:engine/view/selection~Selection|
- *    module:engine/view/documentselection~DocumentSelection|
- *    module:engine/view/position~Position|
- *    Iterable.<module:engine/view/range~Range>|
- *    module:engine/view/range~Range|
- *    module:engine/view/item~Item|
- *    null
- * } module:engine/view/selection~Selectable
  */
 export type Selectable = Selection | DocumentSelection | Position | Iterable<Range> | Range | Item | null;
```

Comment:
Maybe we should not allow `null` here? This probably should be included only in places that use this type?